src/main/java/frc/robot/Constants.java

```diff
@@ -25,31 +33,79 @@
  */
 public final class Constants {
 
-  public static final double ROBOT_MASS = (65) * 0.453592; // 32lbs * kg per pound
-  public static final Matter CHASSIS = new Matter(new Translation3d(0, 0, Units.inchesToMeters(8)), ROBOT_MASS);
-  public static final double LOOP_TIME = 0.13; // s, 20ms + 110ms sprk max velocity lag
+    public static final double ROBOT_MASS = (65) * 0.453592; // 32lbs * kg per pound
```

Comment:
Many of the changes in this file are just whitespace / indentation changes. We could apply uniform formatting/tabbing/indentation rules if we use a formatter, which we can configure to the team's liking. It will also help with things like breaking longer lines of code into multiple lines. Let me know if you want to try that.